index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@700&display=swap" rel="stylesheet"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class="navbar">
+       <p class="nav-item">Rb</p>
+       <img class="nav-item" src="./img/menu.png" alt="the menu image ">
+    </nav>
+    <header>
+        <h1>
+            <p>Hey There.</p>
+            <p>I'm Rachid</p> 
+        </h1>
+        <p class="secondary">I'm a software developer</p>
+        <p class="details">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        
+        </p>
+        <div class="social-media">
+            <a href="https://github.com/benwmx" target="_blank"><img src="./img/002-github-big-logo.png" alt="the image ogf github icon"></a>
+            <a href="https://www.linkedin.com/in/rachid-boudaoudi-1621a0183/"target="_blank"><img src="./img/001-linkedin.png" alt="the image of linkedin icon"></a>
+            <a href="https://twitter.com/benwmx" target="_blank"><img src="./img/003-twitter.png" alt="the image of twitter icon"></a>
+        </div>
```

Comment:
- Your code should be inside a section tag. (1.- The container of the section is a section tag)